apps/desktop/src/components/CommitMessageEditor.svelte

```diff
@@ -68,6 +69,7 @@
 
 	let composer = $state<ReturnType<typeof MessageEditor>>();
 	let titleInput = $state<HTMLTextAreaElement>();
+	const imeHandler = new IMECompositionHandler();
```

Comment:
Let's create an `InjectionToken`for this class, and provide it in `deps.ts`, then we can use `const imeHandler = inject(IME_COMPOSITION_HANDLER)` here?